dataland-inbound-proxy/docker-entrypoint.d/03-configure_robots_file.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+datalandServerName="dataland.com"
+filePath="/var/www/html/robots.txt"
+delimiter="/"
+
+set -exo pipefail
+echo "Letsencrypt path:"$PROXY_LETSENCRYPT_PATH
+fullServerPath=$PROXY_LETSENCRYPT_PATH
+
+serverName="${fullServerPath##*"$delimiter"}"
+echo serverName
+set -u
+
+devText="User-agent: *\nDisallow: /"
+datalandText="User-agent: *\nAllow: / \nDisallow: /keycloak/"
+
+scriptDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$scriptDir"
+cd ..
+
+rm "$filePath"
```

Comment:
If you remove the "original" file anyways, I don't understand why it does even exist in the first place.
Does nginx throw an error during build if it does not exist?

Can't we remove the "robots.txt" file, which is committed in Git and write it from scratch in this script here?